test/internals/bb-spec.ts

```diff
@@ -518,7 +518,7 @@ describe("Interface & initialization", () => {
 		}));
 
 		it("check lazy rendering on callbacks", () => new Promise(done => {
-			const el: any = document.body.querySelector("#chart");
+			const el = <HTMLDivElement>document.body.querySelector("#chart");
```

Comment:
the modern typescript way for that is to use the `document.body.querySelector("#chart") as HTMLDivElement` instead.

Reply:
Thanks for the suggestion.